test/topology_custom/test_gossiper_orphan_remover.py

```diff
@@ -0,0 +1,39 @@
+import time
+import pytest
+import logging
+from test.pylib.manager_client import ManagerClient
+from test.topology.conftest import skip_mode
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_crashed_node_substitution(manager: ManagerClient):
+    servers = await manager.servers_add(3, config={
+        'error_injections_at_startup': ['fast_orphan_removal_fiber']
+    })
+
+    await manager.server_add(config={
+        'error_injections_at_startup': ['crash_before_group0_join']
+    }, expected_error="deliberately crashed for orphan remover test")
+
+    live_eps = await manager.api.client.get_json("/gossiper/endpoint/live", host=servers[0].ip_addr)
+    down_eps = await manager.api.client.get_json("/gossiper/endpoint/down", host=servers[0].ip_addr)
+
+    gossiper_eps = live_eps + down_eps
+    post_crash_servers = await manager.running_servers()
+    server_eps = [s.ip_addr for s in post_crash_servers]
+
+    assert len(gossiper_eps) == (len(server_eps)+1)
```

Comment:
I think this could be flaky, remover fiber races with the test so `down_eps` query might not catch the orphan

Reply:
No actually, since 'speedup_orphan_removal' was not enabled by this point, hence timeout is still 60 secs, so remover fiber can't remove this node before enabling 'speedup_orphan_removal'.

But anyways, to remove dependence on 60s, I have added message injection in new code, so this issue is completely removed now!!

Reply:
> But anyways, to remove dependence on 60s, I have added message injection in new code, so this issue is completely removed now!!

good